docs/best_practices/tool_xml.rst

```diff
@@ -166,7 +166,7 @@ Data parameters
 ---------------
 
 If a (gz)ipped data type is available (e.g. ``fasta.gz`` or ``fastq.gz``) then
```

Comment:
Since you are editing this paragraph, should that be ``(g)zipped`` rather than ``(gz)ipped``?